paths_cli/compiling/networks.py

```diff
@@ -47,34 +89,29 @@ def tis_trans_info(dct):
 
 TPS_NETWORK_PLUGIN = NetworkCompilerPlugin(
     builder=Builder('openpathsampling.TPSNetwork'),
-    parameters=[
-        Parameter('initial_states', compiler_for('volume'),
-                  description="initial states for this transition"),
-        Parameter('final_states', compiler_for('volume'),
-                  description="final states for this transition")
-    ],
-    name='tps'
+    parameters=[INITIAL_STATES_PARAM, FINAL_STATES_PARAM],
+    name='tps',
+    description=("Network for transition path sampling (two state TPS or "
+                 "multiple state TPS)."),
 )
 
 
-MISTIS_NETWORK_PLUGIN = NetworkCompilerPlugin(
-    parameters=[Parameter('trans_info', mistis_trans_info)],
-    builder=Builder('openpathsampling.MISTISNetwork'),
-    name='mistis'
-)
+# MISTIS_NETWORK_PLUGIN = NetworkCompilerPlugin(
+#     parameters=[Parameter('trans_info', mistis_trans_info)],
+#     builder=Builder('openpathsampling.MISTISNetwork'),
+#     name='mistis'
+# )
 
 
-TIS_NETWORK_PLUGIN = NetworkCompilerPlugin(
-    builder=Builder('openpathsampling.MISTISNetwork'),
-    parameters=[Parameter('trans_info', tis_trans_info)],
-    name='tis'
-)
+# TIS_NETWORK_PLUGIN = NetworkCompilerPlugin(
+#     builder=Builder('openpathsampling.MISTISNetwork'),
+#     parameters=[Parameter('trans_info', tis_trans_info)],
+#     name='tis'
+# )
```

Comment:
Are we (temporary) dropping support for these?

Reply:
Yes, these are temporarily disabled. I'm not sure that they were working as desired (these template-function approaches are really nice, but relying only on standard coverage metrics is misleading because you need to test these at integration level, not at unit level). I think it may have handled the input correctly, but it definitely did *not* create reasonable help. So I disabled them until they can be rewritten in a way that both parses the input as desired and provides correct information to the documentation builders (and until there are actual integration tests covering the functionality -- the fact that I could comment this code without raising errors in tests means that it shouldn't be released.)

Reply:
No, this is all within the new features for v0.3, so no concern on that.